sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -584,7 +584,1102 @@
+              "customClass": "header-md",
+              "lockKey": true
+            },
+            {
+              "key": "decreaseInPartnerIncomePanelContent2",
+              "label": "Content",
+              "input": false,
+              "tag": "p",
+              "attrs": [
+                {
+                  "value": "",
+                  "attr": ""
+                }
+              ],
+              "className": "",
+              "content": "Please provide supporting documentation for your request to use current year estimated income for your spouse/common law partner.",
+              "type": "htmlelement",
+              "hideLabel": false,
+              "tags": [],
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+              },
+              "properties": {},
+              "lockKey": true
+            },
+            {
+              "key": "decreaseInPartnerIncomePanelContent3",
+              "label": "Content",
+              "input": false,
+              "tag": "p",
+              "attrs": [
+                {
+                  "value": "",
+                  "attr": ""
+                }
+              ],
+              "className": "",
+              "content": "<h4 class=\"category-header-medium-small\">Provide the following:</h4>\n<ul><li>A letter explaining the circumstances that meet one or more of the Criteria (see above)</li></ul>",
+              "type": "htmlelement",
+              "hideLabel": true,
+              "tags": [],
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+              },
+              "properties": {},
+              "customClass": "align-bullets",
+              "lockKey": true
+            },
+            {
+              "key": "decreaseInPartnerIncomePanelContent4",
+              "label": "Content",
+              "input": false,
+              "tag": "p",
+              "attrs": [
+                {
+                  "value": "",
+                  "attr": ""
+                }
+              ],
+              "className": "",
+              "content": "<h4 class=\"category-header-medium-small\">Provide at least one of the following:</h4>\n<ul><li>All relevant supporting documentation of the circumstance(s) (medical records, letter from the physician, record of employment, layoff or strike, invoices from damages, etc.).</li>\n<li>The more recent CRA Income Tax Return and income information for the year matching the student financial assistance application number (with relevant supporting documentation to show the decrease in income), if available.</li>\n<li>All relevant supporting documentation of estimated total income for the year matching the student financial assistance application number, if applicable (pay stubs, invoices, bank statements, Employment Insurance benefit statements, Record of Employment, etc.).</li>\n</ul>",
```

Comment:
The most recent as per AC.
![image](https://github.com/user-attachments/assets/f9a39f66-e9a3-4143-ab4a-481b9cc2205d)